components/studentPortal/layout/Sidebar.tsx

```diff
@@ -22,45 +31,52 @@ interface SidebarItemProps {
   page: SidebarPage;
   icon: ReactElement;
   notifications?: boolean;
+  userRole: "freemium" | "student" | "coach";
 }
 const SidebarItem = ({
```

Comment:
I think it's time to move sidebarItem into it's own file now

Reply:
did that. good suggestion.  the delete unused imports functionality you implemented works great for this sort of thing.